configutil/go.mod

```diff
@@ -2,9 +2,15 @@ module github.com/hashicorp/go-secure-stdlib/configutil/v2
 
 go 1.20
 
+replace (
+	github.com/hashicorp/go-secure-stdlib/listenerutil => ../listenerutil
+	github.com/hashicorp/go-secure-stdlib/parseutil => ../parseutil
+	github.com/hashicorp/go-secure-stdlib/pluginutil/v2 => ../pluginutil
+)
```

Comment:
Are these left in on purpose?

Reply:
Definitely on purpose, but I'm not sure if they are strictly necessary after merging. Without them, the build seemed to fail because it kept pulling in the previous tag for those libraries.

Reply:
Oh I see now, that makes sense. I guess the alternative would be to split this PR into multiple PRs and merge the updates to dependent libraries once the dependencies have been tagged. I think I have a slight preference for that approach because it makes it easier to reason about exactly what version of the code is being consumed, and also has a better defined solution for making breaking changes in modules. That's also the approach that other modules in the repo consistently take, so it would be good to keep that uniformity. WDYT? cc @jefferai as the repo owner

Reply:
Ah my mistake, you're right about it actually not affecting consumers, from https://go.dev/ref/mod#go-mod-file-replace:

> replace directives only apply in the main module’s go.mod file and are ignored in other modules.

In that case I don't really have a preference, so as it is works for me 👍